libs/error-stack/README.md

```diff
@@ -101,7 +101,7 @@ This will most likely result in an error and print
 
```

Comment:
(comment for line 98)
This path is still valid after the move because we are using a sha to refer to an exact commit, but just pointing out the path in `main` would be incorrect

Reply:
Do we want to move this file into the `.github/assets` folder anyway? https://github.com/hashintel/hash/tree/main/.github/assets (If this is adding weight to `error-stack`, maybe? If it's not, then I guess it's more convenient to leave it inline? Haven't looked at this closely and am doing a brief run-through here)

Reply:
I'd leave it next to `error-stack` as it's very related and generated output form the library. However, I think we should update the link after this PR is merged.